stories/dashboard.stories.js

```diff
@@ -135,7 +135,8 @@ storiesOf( 'Dashboard', module )
 			</WithTestRegistry>
 		);
 	},
-	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } )
+	{ padding: 0,
+	} )
```

Comment:
@danielgent looks instead of removing the `padding` key, you removed the `options` key. Please, add the options back and remove the padding.